### ISSUE TITLE
Enhance the activities bar by dynamically resizing and the bar chart 

### ISSUE THREAD

**Issue by bjamre13** [Amna-Hassan04/Serenity-Guide#30]
issues i found
1.extra space is being utilized in activities bar
2. Barchart is not fit to screen needs scrolling to check the activities.

kindly assign me this task so i can enhance these.

**Comment by bjamre13** [comment]
![Uploading Screenshot 2024-10-02 214103.png…]()

**Comment by Amna-Hassan04** [comment]
@bjamre13 please follow the instructions and attach the relevant screenshots so I can assign you the task.

**Comment by SaiNithinn** [comment]
![Screenshot 2024-10-02 215615](https://github.com/user-attachments/assets/c1fd5c19-4210-4184-86ac-f6331c28d010)

**Comment by bjamre13** [comment]
![IMG-20241002-WA0002.jpg](https://github.com/user-attachments/assets/58239cc7-4736-4f13-99a4-306b6ed84ee5)

**Comment by bjamre13** [comment]
![IMG-20241002-WA0002.jpg](https://github.com/user-attachments/assets/58239cc7-4736-4f13-99a4-306b6ed84ee5)

Upon opening the bar graph it's not fitting in the window screen

**Comment by Amna-Hassan04** [comment]
rigth so what are u going to implement here? still confused

**Comment by bjamre13** [comment]
I am currently working on resolving this issue. Please allow me some time to finalize the details. I will provide a detailed format of the bugs by 10 PM at the latest.
Please hold this issue to me.!!

Thank you for your understanding.

**Comment by bjamre13** [comment]
![IMG-20241003-WA0005.jpg](https://github.com/user-attachments/assets/fc4cd1fe-d13a-4df0-8811-3da3cc03b240)

When we try to open the bar chart in full screen mode... it's gui not that users expected and upon using the auto scaling option it has no change.
I really want to contribute to this issue and resolve it to my best expectations.

**Comment by bjamre13** [comment]
https://github.com/user-attachments/assets/336a729f-e06d-4d72-b06c-560d96da8366

This is the video of the issue I'm talking about.!!
Hope you get it
Thank you.

**Comment by bjamre13** [comment]
@Amna-Hassan04  kindly review the issue.

**Comment by Amna-Hassan04** [comment]
@bjamre13 yes I see the problem now, you can work on this, and make sure to use the latest version of the project.

**Comment by bjamre13** [comment]
Could you please assign the labels as well of , GSSoc ext ,level and hactoberfest.
Thank you.
